src/pages/LogOutPreviousUserPage.js

```diff
@@ -33,10 +35,12 @@ const defaultProps = {
 };
 
 function LogOutPreviousUserPage(props) {
+    const initUrl = useContext(InitialUrlContext);
     useEffect(() => {
-        Linking.getInitialURL().then((transitionURL) => {
+        Linking.getInitialURL().then((url) => {
             const sessionEmail = props.session.email;
-            const isLoggingInAsNewUser = SessionUtils.isLoggingInAsNewUser(transitionURL, sessionEmail);
+            const transitionUrl = NativeModules.ReactNativeModule ? CONST.DEEPLINK_BASE_URL + initUrl : url;
```

Comment:
Are you sure that `ReactNativeModule` is the best name for this new module? Maybe something more descriptive would be a better option?

Reply:
That is true, it was named this way looking from OldDot perspective, where React Native is a new addition. I believe we could rename it eg. `HybridAppModule`